Applications/PaintBrush/PaintableWidget.h

```diff
@@ -29,6 +29,8 @@ class PaintableWidget final : public GWidget {
 
     Function<void(Color)> on_primary_color_change;
     Function<void(Color)> on_secondary_color_change;
+    
+    bool eraser_use_secondary_color { true };
```

Comment:
This behavior should be a property of the `EraseTool`, not of the `PaintableWidget`.
Also, please don't add public member variables like. Instead, add a private variable and expose it through a getter & setter.